meta/resources/onboarding.md

```diff
@@ -56,6 +56,19 @@ to reach out to you!
 If you need access to the Figma organization as a designer, please reach out the Design Lead (reach out to [Matthew](https://github.com/purduehackers/dark-forest/blob/main/people/organizers/hewillyeah.md)
 as we're in the process of getting a new Design Lead).
 
+### Access to Purdue Hackers Office (Optional)
+
+If you are a member of the Events Divison or otherwise need access to the Lawson Office where materials such as the [Hack Cart](/events/hack-night/hack-cart)
```

Comment:
```suggestion
If you need access to the Lawson Office, where materials such as the [Hack Cart](/events/hack-night/hack-cart)
```

this is probably going to be a point of contention forever but i really don't want to reference things like this. being a member of the events division does not, entitle one to anything; working on an event that requires access to the office does. to us in february 2025, these two are equivalent, but only because we understand that "being part of the events division" is a function of the work you're doing. but when we mention identity like this, Someone who Is Part of Some Division, that understanding can get lost, and too much of this is how we end up with people who don't do anything having legit-sounding roles.